### ISSUE TITLE
Sign-in button missing visiting a profile page in portrait mode

### ISSUE THREAD

**Issue by Eidan78** [freeCodeCamp/freeCodeCamp#46534]
On a mobile browser or on a desktop one (under a width resolution of 456px ) there is no sign-in button to login in freecodecamp even in the burger menu in portrait mode.

**To Reproduce**
1. Visit a profile page from the smartphone in portrait mode (Ex. [this page](https://www.freecodecamp.org/eidan))
2. Log out if logged in
3. The sign-in button is missing on the page and in the burger menu

**Expected behavior**
The sign-in button should be visible on the main page and in the burger menu or at least in the burger menu

**Screenshots**
![image](https://user-images.githubusercontent.com/8456092/174251683-5c0c40d0-b398-404e-b254-005e380e92e2.png)

**Desktop**
 - OS: any
 - Browser: any
 - Version: any

**Smartphone**
 - Device: any
 - OS: any
 - Browser: any
 - Version: any

**Comment by ahmadabdolsaheb** [comment]
It does not show up for that specific device size. You should be able to see it if you tilt your device.
The best solution is to minimize the logo early so the sign in button could be displayed.

**Comment by Eidan78** [comment]
> It does not show up for that specific device size. You should be able to see it if you tilt your device. The best solution is to minimize the logo early so the sign in button could be displayed.

I agree with reducing the logo size, I edited the post with your suggestion. That's why I specified that it doesn't works in protrait mode but it does in landscape.

**Comment by ahmadabdolsaheb** [comment]
After a certain break point, we would need to use icons for the menu and the sign in button.

**Comment by shootermv** [comment]
i can see 'sign-in to change the theme' option at open menu, but for some reason it is disabled
![image](https://user-images.githubusercontent.com/1336862/174324351-b0bd0956-4643-4a5d-b0da-eef9079e07d8.png)

**Comment by ahmadabdolsaheb** [comment]
That is the theme toggle (light/dark) but you have to be signed in to access that.

**Comment by shootermv** [comment]
So, can 'sign_in' button be also on the opened menu?

**Comment by ahmadabdolsaheb** [comment]
The position of the sign in button should be next to the menu button in the same order it is displayed on larger screens.

**Comment by shootermv** [comment]
Do you think this looks good?
![image](https://user-images.githubusercontent.com/1336862/174473707-b2da2d38-71e1-4087-b6f3-88c3bdc65527.png)

**Comment by ahmadabdolsaheb** [comment]
Just had a look at our localized versions. It seems like we need to show icons for instead of 'Menu' and 'Sign in' under 600px. 

Feel free to work on the logic to replace the text with icons. Once ready, I will find the proper icon replacement.

**Comment by shootermv** [comment]
how about following icons?
![image](https://user-images.githubusercontent.com/1336862/175254695-8d670ad8-7e24-4f0c-a571-0a922135cc64.png)

**Comment by shootermv** [comment]
pushed
